docs/projects/load-project-yaml.ipynb

```diff
@@ -0,0 +1,1359 @@
+{
```

Comment:
<ol><li><a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#load-or-create-a-new-project-based-on-an-existing-project-yaml-file" target="_blank">Load </a>a project from remote URL</li><li><a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#get-the-function-object" target="_blank">Get the function object</a> - Get a function object</li><li><a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#run-the-function" target="_blank">Run the function</a> - Run project functions</li><li><a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#build-the-function" target="_blank">Build the function</a> - Build project functions</li><li><a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#deploy-the-function" target="_blank">Deploy the function</a> - Deploy project functions</li><li><a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#run-the-workflow" target="_blank">Run the workflow</a> - Run project workflow</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>fixed</p>